src/components/StartScriptGenerator.tsx

```diff
@@ -0,0 +1,250 @@
+import React, { useEffect, useState, useRef } from 'react';
+import '@site/src/css/start-script-generator.css';
+
+const markerPoints = [4, 8, 12, 16, 20];
+
+type FlagType = {
+    label: string;
+    value: string;
+    description: string;
+};
+
+const WINDOWS_AUTO_RESTART = ":start\n%%CONTENT%%\n\necho Server restarting...\necho Press CTRL + C to stop.\ngoto :start";
+const LINUX_AUTO_RESTART = "while [ true ]; do\n    %%CONTENT%%\n    echo Server restarting...\n    echo Press CTRL + C to stop.\ndone";
+
+const FLAGS: { [key: string]: FlagType } = {
+    AIKARS: {
+        label: 'Aikar\'s',
```

Comment:
I hate how inconsistent the usage of " and ' is in this file, do we have editor config or eslint or prettier or whatever to enforce one? lol

Reply:
Haha, yeah. I've changed this file for now. In the future id like to have some sort of formatting checker and also a British checker for me :P